### DISCUSSION TITLE
Prüfung auf Basis der Consent Policies

### DISCUSSION THREAD

**Discussion by SebStaeubert** [medizininformatik-initiative/fts-next#464] [Ideas]
Ziel ist es eine Berücksichtigung des im MII BC dokumentierten Patientenwillen zu implementieren, d.h. ein Prüfung der MII BC Policies entspr. der Vorgaben der TF CU.

Eine Umsetzung ist im FDPG erfolgt. Vorschlag: die FTSnext setzen dies analog um.
Siehe folgende Tickets:

- https://github.com/medizininformatik-initiative/torch/issues/6
- https://github.com/medizininformatik-initiative/kerndatensatzmodul-consent/issues/52

**Comment by knoppiks** [comment]
- Everything Data Selector verwendet bereits Datums-Einschränkung
- FHIR Server darf aber prinzipiell mehr zurückliefern
- Einschränkung nach Selektion notwendig?
- Kann das der Deidentifhir?
